platforms/ios/framework/src/TGMapView.h

```diff
@@ -225,7 +225,7 @@ TG_EXPORT
 /**
  Move the map camera to a new position with an animation that pans and zooms in a smooth arc.
 
- The animation duration is calculated based on the distance to the new camera position.
+ The animation duration is calculated based on the distance to the new camera position assuming a speed of 1.0 (mercator meters per second)
```

Comment:
Not quite accurate - `speed` is a unit-less scaling factor on the calculated duration. No need to mention speed in this context.